src/DynamoCore/Configuration/IPreferences.cs

```diff
@@ -128,6 +128,8 @@ public interface IPreferences
         /// </summary>
         bool ShowEdges { get; set; }
 
+        List<string> NamespacesToExcludeFromLibrary { get; set; }
```

Comment:
I just submitted this PR, (awaiting a review from @sharadkjaiswal ) for a similar issue and possible solution:
https://github.com/DynamoDS/Dynamo/pull/6726